transport/graphsync/graphsync.go

```diff
@@ -448,6 +449,51 @@ func (t *Transport) gsOutgoingBlockHook(p peer.ID, request graphsync.RequestData
 	}
 }
 
+// gsReqRecdHook is called when graphsync receives an incoming request for data
+func (t *Transport) gsReqQueuedHook(p peer.ID, request graphsync.RequestData) {
+	msg, err := extension.GetTransferData(request, t.supportedExtensions)
+	if err != nil {
+		return
+	}
+	// extension not found; probably not our request.
+	if msg == nil {
+		return
+	}
+
+	var chid datatransfer.ChannelID
+	var ch *dtChannel
+	if msg.IsRequest() {
+		// when a data transfer request comes in on graphsync, the remote peer
+		// initiated a pull
+		chid = datatransfer.ChannelID{ID: msg.TransferID(), Initiator: p, Responder: t.peerID}
+		// Lock the channel for the duration of this method
+		ch = t.trackDTChannel(chid)
+		ch.lk.Lock()
+		defer ch.lk.Unlock()
+
+		request := msg.(datatransfer.Request)
+		if request.IsRestart() || request.IsNew() {
+			log.Infof("channelID=%s, pull request queued, req=%+v", chid, request)
```

Comment:
For consistency with the other logs in this file
```suggestion
			log.Infof("%s: pull request queued, req=%+v", chid, request)
```